terraform/modules/prometheus_all/input.tf

```
@@ -1,36 +1,17 @@
+variable monitoring_instance_name {}
 variable monitoring_org_name {}
-
 variable monitoring_space_name {}
 
-variable paas_user {}
-
-variable paas_password {}
-
-variable paas_sso_code {}
-
-variable grafana_admin_password {}
+variable paas_exporter_username {}
+variable paas_exporter_password {}
 
 variable alertmanager_config { default = "" }
-
 variable alert_rules { default = "" }
 
-variable paas_exporter_config {
-  type = map
-  default = {
-    API_ENDPOINT = ""
-    USERNAME     = ""
-    PASSWORD     = ""
-  }
-}
-
-variable grafana_config {
-  type = map
-  default = {
-    dashboard_directory  = "../grafana/dashboards"
-    datasource_directory = "../grafana/datasources"
-    configuration_file   = "../grafana/config/grafana.ini"
-  }
-}
+variable grafana_google_client_id { default = "" }
+variable grafana_google_client_secret { default = "" }
+variable grafana_admin_password {}
+variable grafana_json_dashboards {}
```

```suggestion
variable grafana_json_dashboards { default = [] }
```
might be worth having this empty when creating an fresh instance and then creating a dashboard from the UI and exporting as a JSON (to be used in subsequent runs).

---

Good point, thanks